platform/x11/context_gl_x11.cpp

```diff
@@ -191,7 +191,8 @@ Error ContextGL_X11::initialize() {
 
 	swa.colormap = XCreateColormap(x11_display, RootWindow(x11_display, vi->screen), vi->visual, AllocNone);
 	x11_window = XCreateWindow(x11_display, RootWindow(x11_display, vi->screen), 0, 0, OS::get_singleton()->get_video_mode().width, OS::get_singleton()->get_video_mode().height, 0, vi->depth, InputOutput, vi->visual, valuemask, &swa);
-
+	XStoreName(x11_display, x11_window, "Godot Engine");
```

Comment:
Forgive my lack of knowledge, but doesn't this mean that exported applications would also show up as "Godot Engine" on the taskbar?

Reply:
XStoreName() is also called each time the title of window is changed and therefore will be overwritten immediately (I checked). But not calling it at this point causes the issue...

Reply:
Ah, gotcha. Thanks for explaining!

Reply:
But you made me curious for the Window-Preview. So I checked. Here it doesn't update without a workspace switch... But that's on my old Cinnamon. Someone with newer version or with Gnome needs to test this...

![godot_executable](https://user-images.githubusercontent.com/6639237/56902049-c0e0a880-6a99-11e9-936d-ad43904816c6.png)